### ISSUE TITLE
Public descriptions and notes?

### ISSUE THREAD

**Issue by geekedtv** [Kovah/LinkAce#479]
I'm having a hard time understanding why link descriptions are not shown on the public pages when I log out and look at my links. Also notes have an option to make private but I can't see the notes when logged out either. Is this something planned? I'd like public links to have more context so people know more about them and would like the description of the links to be shown.

**Comment by Kovah** [comment]
I agree, the current state of how links and everything else is displayed - both to users and guests - is quite a mess. It will be completely redone in version 2.0.

**Comment by geekedtv** [comment]
Awesome. Sorry if this was already brought up. It’s a beautiful app just needs a few tweaks!
